src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+
+## Futuro
+
+Immergiamoci in un paio di hints sperimentali. Molto vicino al rilascio abbiamo i hint prioritari(Priority hints), che vengono attivamente sperimentati nella comunità web. Abbiamo anche i 103 Early Hints in HTTP/2, che è ancora all'inizio e ci sono alcuni giocatori come [Chrome e Fastly che collaborano per i prossimi test di prova](https://www.fastly.com/blog/beyond-server-push-experimenting-with-the-103-early-hints-status-code).
+
+### Hints prioritari
+
+[Priority hints](https://developers.google.com/web/updates/2019/02/priority-hints) sono un'API per esprimere la priorità di fetch di una risorsa: alta, bassa o automatica. Possono essere utilizzati per ridurre la priorità delle immagini (ad esempio all'interno di un Carousel), ridefinire la priorità degli script e persino aiutare a ridurre la priorità dei fetch.
+
+Questo nuovo hint può essere utilizzato come tag HTML o modificando la priorità delle richieste di fetch tramite l'opzione `importance`, che assume gli stessi valori dell'attributo HTML.
+
+```html
+<link rel="preload" href="/js/script.js" as="script" importance="low">
+
+<img src="/img/in_view_but_not_important.svg" importance="low" alt="I'm not important!">
+```
+
+Con `preload` e `prefetch`, la priorità è impostata dal browser a seconda del tipo di risorsa. Utilizzando le Priority Hints possiamo forzare il browser a modificare l'opzione predefinita.
+
+{{ figure_markup(
+  caption="La percentuale di adozione del Priority Hint sui dispositivi mobile.",
+  content="0.77%",
+  classes="big-number",
+  sheets_gid="1596669035",
+  sql_file="priority_hints.sql"
+) }}
+
+Finora solo lo 0,77% dei siti web ha adottato questo nuovo hint poiché Chrome sta ancora [attivamente](https://www.chromestatus.com/features/5273474901737472) sperimentando e al momento del rilascio di questo articolo la feature è in attesa.
+
+L'utilizzo maggiore è con elementi di script, il che non sorprende poiché il numero di file JS primari e di terze parti continua a crescere.
+
+{{ figure_markup(
+  caption="La percentuale di risorse nei dispositivi mobile con un hint che utilizza la \"low\" priority.",
+  content="16%",
+  classes="big-number",
+  sheets_gid="1098063134",
+  sql_file="priority_hints_by_importance.sql"
+) }}
+
+I dati ci mostrano che l'83% delle risorse che utilizzano le Priority Hints utilizza una priorità "alta" sui dispositivi mobile, ma qualcosa a cui dovremmo prestare ancora più attenzione è il 16% delle risorse con priorità "bassa".
+
+Priority hints hanno un chiaro vantaggio come strumento per prevenire il caricamento dispendioso tramite la priorità "bassa" aiutando il browser a decidere cosa togliere la priorità e restituendo CPU e larghezza di banda significative per completare prima le richieste critiche, piuttosto che come tattica per provare a ottenere risorse caricate più rapidamente con la priorità "alta".
+
+### 103 Early Hints in HTTP/2
```

Comment:
Can we translate this? Or maybe only the "in"?

Reply:
@chefleo did you see this? Hidden under a "Load more..." so you might have missed it.

Reply:
So interesting enough the way the word `in` is used here, is exactly the same in both English and Italian. So I think we are good here.

Reply:
Cool. Resolving.